File: 800/59A/59A.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
    string s;

    cin >> s;

    int lower = 0;
    int upper = 0;
    
    for(char c : s){
        if(islower(c)){
            lower++;
        }
        else{
            upper++;
        }
    }

    if(lower > upper){
        for(char &c : s){
            c = tolower(c);
        }
    }
    else if(lower < upper){
        for(char &c : s){
            c = toupper(c);
        }
    }
    else{
        for(char &c : s){
            c = tolower(c);
        }
    }

    cout << s << endl;

    return 0;
}

File: 59A/59A.py
s = input()

lower = 0
upper = 0

for c in s:
    if c.islower():
        lower += 1
    if c.isupper():
        upper += 1
        
if lower > upper:
    print(s.lower())
elif lower < upper:
    print(s.upper())
else:
    print(s.lower())